debugview/src/debug/kotlin/com/revenuecat/purchases/debugview/models/InternalDebugRevenueCatScreenViewModel.kt

```diff
@@ -0,0 +1,104 @@
+        SettingScreenState.NotConfigured(configurationGroup()),
+    )
+
+    init {
+        if (Purchases.isConfigured) {
+            refreshInfo()
+        }
+    }
+
+    private fun refreshInfo() {
+        viewModelScope.launch {
+            try {
+                val offerings = Purchases.sharedInstance.awaitOfferings()
+                val customerInfo = Purchases.sharedInstance.awaitCustomerInfo()
+                _state.update {
+                    SettingScreenState.Configured(
+                        configurationGroup(),
+                        customerInfoGroup(customerInfo),
+                        offeringsGroup(offerings),
+                    )
+                }
+            } catch (e: PurchasesException) {
+                Log.e("RevenueCatDebugView", "Error getting RevenueCat SDK info for debug view. Exception: $e")
+            }
+        }
+    }
+
+    private fun configurationGroup(): SettingGroupState {
+        val storeName = if (Purchases.isConfigured) {
+            Purchases.sharedInstance.store.name
+        } else {
+            "Not configured"
+        }
+        val observerMode = if (Purchases.isConfigured) {
+            "${!Purchases.sharedInstance.finishTransactions}"
+        } else {
+            "Not configured"
+        }
+        return SettingGroupState(
+            "Configuration",
+            listOf(
+                SettingState.Text("SDK version", Purchases.frameworkVersion),
+                SettingState.Text("Is configured", "${Purchases.isConfigured}"),
+                SettingState.Text("Store", storeName),
+                SettingState.Text("Observer mode", observerMode),
+            ),
+        )
+    }
+
+    private fun customerInfoGroup(customerInfo: CustomerInfo): SettingGroupState {
+        return SettingGroupState(
+            title = "Customer info",
+            settings = listOf(
+                SettingState.Text("Current User ID", Purchases.sharedInstance.appUserID),
+                SettingState.Text("Original User ID", customerInfo.originalAppUserId),
+                SettingState.Text(
+                    "Active entitlements",
+                    customerInfo.entitlements.active
+                        .map { "${it.key} until ${it.value.expirationDate}" }
+                        .joinToString("\n")
+                        .takeIf { it.isNotEmpty() } ?: "None",
+                ),
+                SettingState.Text("Verification result", customerInfo.entitlements.verification.name),
+                SettingState.Text("Request date", customerInfo.requestDate.toString()),
+            ),
+        )
+    }
+
+    private fun offeringsGroup(offerings: Offerings): SettingGroupState {
+        return SettingGroupState(
+            title = "Offerings",
+            settings = offerings.all.values.map { offering ->
+                SettingState.Text(
+                    title = offering.identifier,
+                    content = "TODO",
```

Comment:
Still need to make offerings visible. My idea is to display them as alerts in a future PR. Navigation within bottom sheets requires additional experimental libraries on Google's side, so I wanted to avoid that for now.